src/main/java/org/broadinstitute/hellbender/transformers/IUPACReadTransformer.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.hellbender.transformers;
+
+import org.broadinstitute.hellbender.utils.BaseUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+public class IUPACReadTransformer implements ReadTransformer {
+    private static final long serialVersionUID = 1L;
+    private boolean strictMode;
+
+    public IUPACReadTransformer() {
+        this.strictMode = false;
+    }
+
+    public IUPACReadTransformer(final boolean strictMode) {
+        this.strictMode = strictMode;
+    }
+
+    @Override
+    public GATKRead apply(GATKRead read) {
+        read.setBases(BaseUtils.convertIUPACtoN(read.getBases(), strictMode, false));
```

Comment:
You can take this or leave it. I actually think it might be useful to attach a one-shot-logger to this that warns the user that this has happened for some of their reads and spits out the readname so they can check later. Given how this is set up it might be a bit of a hassle though...

Reply:
That's a really good point.  I have something I'm happy enough with, but let me know if you can think of something more efficient.